### DISCUSSION TITLE
Add UUID Compatibility for Table Primary Keys

### DISCUSSION THREAD

**Discussion by bayukartiko** [ash-jc-allen/short-url#282] [Q&A]
In the current database migration setup, the package uses `$table->bigIncrements('id');` to define the primary key. I would like to change this to use UUIDs instead, specifically in PostgreSQL: `$table->uuid('id')->default(DB::raw('gen_random_uuid()'))->primary()`.

I'm using `Laravel v11.8` and `short-url v8.0.0`.

is it possible?

**Comment by ash-jc-allen** [comment]
Hey @bayukartiko, that's a great question. At the moment, I've not got any plans to add any support for UUIDs in the package, sorry. But if it's something enough people want, I'd be open to looking into it 😄
